__mocks__/forms/afe-forms/test-schema-transformer-form.json

```diff
@@ -19,6 +23,7 @@
               "label": "Numeric",
               "id": "dem_numeric",
               "type": "obs",
+              "required": "false",
```

Comment:
Curious about the need for this. Isn't this by default `false`?

Reply:
This is to test out that after the schema transformation we end up with a boolean